### PR TITLE
DB.SheduleField support

### PR DESCRIPTION
**Pull request by ejs-ejs**
Partial support for pickling DB.ScheduleField.

### CODE REVIEW

File: pyrevitlib/pyrevit/revit/db/pickling.py [hunk 1]
```diff
@@ -150,6 +150,76 @@ class ViewType(EnumSerializable):
...
+        return self.name, self.heading, self.hidden, self.width.deserialize(), self.FO, self.FO_Accuracy.deserialize(), self.FO_sym, self.FO_usePlus
+      else:
+        return self.name, self.heading, self.hidden, self.width.deserialize()
+
+
+class Grid(Serializable):
+    api_types = DB.Grid
+
+    def __init__(self, gridline):
+        from rpw import doc
```
Comment (jmcouffin) [inline pyrevitlib/pyrevit/revit/db/pickling.py:179]: Use 'doc = revit.doc' from the pyRevit revit module?

Comment (ejs-ejs) [reply pyrevitlib/pyrevit/revit/db/pickling.py:179]: the Grid class is already in the master, so it may be skipped.

**Comment by eirannejad** [comment]
This PR seems odd. `DB.ScheduleField` does not have any constructors. How are you `deserialize`ing the stored data into a `DB.ScheduleField`?

**Comment by ejs-ejs** [comment]
> This PR seems odd. `DB.ScheduleField` does not have any constructors. How are you `deserialize`ing the stored data into a `DB.ScheduleField`?

The deserializing is needed for numeric fields, like 'width' AFAIR . What I [think | hope], the API is creating a new DB.ScheduleField and the properties can be applied to the existing DB.ScheduleField. I need to clean the kind-of-working code, the repo is  https://github.com/ejs-ejs/py-ejs.

**Comment by jmcouffin** [comment]
@ejs-ejs did you get the chance to validate this change and test it?

**Comment by jmcouffin** [comment]
I will leave it alive for an extra week then close the PR

**Pull request closed without merging**